src/server/game/Spells/SpellMgr.cpp

```diff
@@ -7056,6 +7056,29 @@ void SpellMgr::LoadDbcDataCorrections()
         spellInfo->rangeIndex = 6; // 100yd
     });
 
+    // ///////////////////////////////////////////
+    // ///////////OPERATION GNOMERGAN/////////////
+    // ///////////////////////////////////////////
+    ApplySpellFix({ 51590 }, [](SpellEntry* spellInfo)
+    {
+        spellInfo->MaxAffectedTargets = 1;
+    });
+
+    ApplySpellFix({ 51590 }, [](SpellEntry* spellInfo)
+    {
+        for (int8 i = 0; i < 3; ++i)
+            spellInfo->EffectImplicitTargetB[i] = TARGET_UNIT_TARGET_ANY;
+    });
+
+    ApplySpellFix({
+        75536, // Explosion (prevent error message in console)
+        71607, // Explosion (prevent error message in console)
+        71610  // Emergency Recall [Camera trigger]
+        }, [](SpellEntry* spellInfo)
+    {
+        spellInfo->EffectImplicitTargetB[0] = TARGET_UNIT_TARGET_ANY;
+    });
```

Comment:
I don't know, you can help to decide.

But I think we could put these in already existing ApplySpellFixes

Reply:
@FALL1N1 what do u think?

Reply:
In my opinion it's better to be split away, that way it's easy to navigate and search what you need for, but yeah, it's a matter of opinions, I really don't mind both ways.

Reply:
Then we can keep it :)